_includes/homepage-container_first-band.html

```diff
@@ -0,0 +1,13 @@
+<div class="full-width-bg component homepage-content-band">
+  <div class="grid-wrapper">
+    <div class="width-12-12">
+      <h2>Container First</h2>
+    </div>
+    <div class="width-12-12 block-content">
+      <p>Quarkus tailors your application for GraalVM and HotSpot. Amazingly fast boot time, incredibly low RSS memory (not just heap size!) offering near instant scale up and high density memory utilization in container orchestration platforms like Kubernetes. We use a technique we call compile time boot. <a href="/vision/container-first">Learn more.</a></p>
+      <pre class="highlightjs highlight"><code class="bash">$ ./my-native-java-rest-app
+Quarkus started in 0.008s</code></pre>
+      <img src="{{site.baseurl}}/assets/images/homepage_container_first.png">
```

Comment:
The old style was much more catchy and recognizable - the new one is very "corporate" and my eye instinctually just gloss over it.

Could we keep the old terminal/matrix style but choose one of the quarkus colors instead ?

Reply:
CC @jkbrock ^